### ISSUE TITLE
Winnow Update

### ISSUE THREAD

**Issue by jadelintott** [ohtap/ohtap#15]
### Summary

Need to update Winnow so that it works with our new metadata set-up
Will update both here and in the read me as more detail emmerges

### Actions

- [ ] Set up meeting with Natalie
- [ ] Do detailed look over Winnow code
- [ ] Add Natalie Sada to this ticket once her github is added

**Comment by kmcdono2** [comment]
@jadelintott I know this is on pause, but can you document how far you got with this for me here?

**Comment by jadelintott** [comment]
Absolutely. We made a lot of Winnow fixes and fixed a lot of bugs. To me, the next steps feel like fixing the gender filtering especially when Winnow displays statistics, figuring out how to get output from Winnow marking false hits, and of course getting it to work on Windows
